dbms/sqlquery/mysql/src/cloud/benchflow/monitors/querymysql.go

```diff
@@ -35,35 +36,40 @@ func queryHandler(w http.ResponseWriter, r *http.Request) {
     }
     // Checks over the rows if the comparison holds
     rowI := 0
-    response := Response{false}
+    result := false
+    queryResponse := ""
     for rows.Next() {
```

Comment:
@VincenzoFerme The assumption is already that the response should be a single row with a single value (like a count query), because managing completely arbitrary queries can be a mess to manage.